skietbaan_fe/src/components/List.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import { Table } from "react-bootstrap";
+import '../components/ListStyle.css'
+class List extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      posts :[]
+    }
+  }
+  componentWillMount() {
+    fetch('http://skietbaan.retrotest.co.za/api/User')
```

Comment:
Seeing this fetch copy pasted multiple times.... pull into a method you can import wherever you need to use it.